src/components/ApplicationDetails/tabs/ApplicationOverviewTab.tsx

```diff
@@ -1,16 +1,67 @@
 import * as React from 'react';
+import componentsIcon from '../../../imgs/illustrations/Components.svg';
+import editCodeIcon from '../../../imgs/illustrations/Edit code.svg';
+import githubAppIcon from '../../../imgs/illustrations/Github app.svg';
+import pipelineIcon from '../../../imgs/illustrations/Pipeline.svg';
 import AppRecentCommits from '../../Commits/AppRecentCommits';
+import WhatsNextSection, { WhatsNextItem } from '../../WhatsNext/WhatsNextSection';
 import AppWorkflowSection from './overview/sections/AppWorkflowSection';
 
 type ApplicationOverviewTabProps = {
   applicationName: string;
 };
 
 const ApplicationOverviewTab: React.FC<ApplicationOverviewTabProps> = ({ applicationName }) => {
+  const whatsNextItems: WhatsNextItem[] = [
+    {
+      title: 'Add a component',
+      description: 'Grow your application by adding components.',
+      icon: componentsIcon,
+      cta: {
+        label: 'Add component',
+        href: `/stonesoup/import?application=${applicationName}`,
+      },
+      helpId: 'add-component',
+    },
+    {
+      title: 'Install our GitHub app',
+      description: 'Install the GitHub app to monitor your work from a commit to deployment.',
+      icon: githubAppIcon,
+      cta: {
+        label: 'Start the flow',
+        href: 'https://github.com/apps/appstudio-staging-ci',
+        external: true,
+      },
+      helpId: 'add-component',
```

Comment:
Do we need to hide the learn more link for ids/section that don't have content?

Reply:
We'll be getting the learn more ids for all the whats next items from docs team. I've used `add-component` for now just to show how it looks.